moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -245,6 +262,12 @@ const MealReservation = (): JSX.Element => {
                             : 'mealReservation__container__restaurantTitleWrapper mealReservation__container__restaurantTitleWrapper--empty'
                     }
                 >
+                    <DisabledReservationModal
+                        modalIsOpen={showDisabledReservation}
+                        closeModal={() => setShowDisabledReservation(!true)}
```

Comment:
why would we negate `true` where there is `false`
`closeModal={() => setShowDisabledReservation(false)}`